recipe/meta.yaml

```diff
@@ -56,15 +54,17 @@ test:
     - pyarrow.compute
```

Comment:
```suggestion
    - pyarrow.acero
    - pyarrow.compute
```

Should this be added since the option for acero is set to on [here](https://github.com/AnacondaRecipes/pyarrow-feedstock/pull/19/files#diff-54cf74e113dd3f6d11e092fdb1d888ec82c69bdafbb15cfb6570c83ecad28f33R6)?

Reply:
> Same thing with `pyarrow.dataset_parquet`?

Reply:
And `pyarrow.parquet_encryption`?

Reply:
And `pyarrow.substrait`?

Reply:
addressed, thank you.